src/Test/Utilities/Portable/FX/DumpProcRunner.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities
+{
+    public class ProcDumpRunner
```

Comment:
I'm beginning to wonder if we should go through the trouble of sharing the code here. It's a very small function and the requirements of the two evironments differ heavily: VS wants to use ENV vars because it's pretty much the only choice it has while RunTests prefers explicit passing.